README.md

```diff
@@ -1 +1,131 @@
-# morse-decoder
\ No newline at end of file
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Morse Decoder <a name="about-project"></a>
+
+**Morse Decoder** is a project that involves reading in a Morse Code string and translating it to English using Ruby
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Function to input morse code messages and translate them to English
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This project is useful and important as it gives developers an experience of how to use databases
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need a compatible browser to load the html file
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/aamir-asaram/morse-decoder.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd morse-decoder
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Aamir Asaram**
+
+- GitHub: [@aamir-asaram](https://github.com/aamir-asaram)
+- Twitter: [@aaaaamir\_\_](https://twitter.com/aaaaamir__)
+- LinkedIn: [Aamir Asaram](https://www.linkedin.com/in/aamir-asaram/)
```

Comment:
Please fix the broken `Twitter` link, let's avoid broken links in our readme for a professional repository. 🤗